examples/workflow-test/src/graph/elements/label-heading.po.ts

```diff
@@ -0,0 +1,9 @@
+import { Clickable, Mix, Renameable } from '@eclipse-glsp/glsp-playwright-core/extension';
+import { ModelElementMetadata, PLabel } from '@eclipse-glsp/glsp-playwright-core/glsp';
+
+export const Base = Mix(PLabel).flow(Clickable).flow(Renameable).build();
```

Comment:
I'd rather this be called `LabelHeadingMixin` to be consistent with the other POs.